src/test/java/com/issoft/training/tests/AutomationPracticeShoppingTests.java

```diff
@@ -0,0 +1,188 @@
+        // clear My Wishlist
+        Assert.assertTrue(myWishlistsPage.isDisplayedDeleteMyWishlistBtn());
+        myWishlistsPage.clickDeleteMyWishlistBtn();
+        Alert alert = driver.switchTo().alert();
+        alert.accept();
+        driver.navigate().refresh();
+    }
+
+    @Test( priority = 3, groups = {"wishlistTests", "checkinTests"},
+            description = "Verify the ability to add to my Wishlist.")
+    @Severity(SeverityLevel.NORMAL)
+    @Description("Test Description: Verify the ability to add to my Wishlist.")
+    @Story("Story name: Add a product to Wishlist.")
+    public void AddProductToWishlistTest() {
+        // test AP-4
+        myAccountPage = new MyAccountPage(driver);
+        myWishlistsPage = new MyWishlistsPage(driver);
+
+        driver.navigate().to(getProperty("url_my_wishlist_page"));
+        Assert.assertTrue(myWishlistsPage.isDisplayedMyWishListHeader());
+
+        // create Wishlist
+        myWishlistsPage.inputWishListName(Constants.MY_WISHLIST_NAME);
+        myWishlistsPage.clickSubmitWishlistBtn();
+        Assert.assertTrue(myWishlistsPage.isDisplayedMyWishListByName());
+
+        myWishlistsPage.clickFirstProductDetailsLink();
+        Assert.assertTrue(myWishlistsPage.isDisplayedMyWishListBtn());
+
+        // Expected result: Product was added to your Wishlist
+        myWishlistsPage.clickMyWishListBtn();
+        Assert.assertTrue(myWishlistsPage.isDisplayedProductAddedAlert());
+
+        // verify My Wishlist
+        myWishlistsPage.clickCloseProductAddedAlert();
+        driver.navigate().to(getProperty("url_my_wishlist_page"));
+        Assert.assertTrue(myWishlistsPage.isDisplayedMyWishListByName());
+    }
+
+
+    @Test( priority = 4, groups = {"cartTests", "checkinTests"},
+            description = "Verify the ability to add to Cart.")
+    @Severity(SeverityLevel.NORMAL)
+    @Description("Test Description: Verify the ability to add to Cart.")
+    @Story("Story name: Add products to Cart.")
+    public void AddProductsToCart() {
+        // test AP-5
+        myAccountPage = new MyAccountPage(driver);
+        myWishlistsPage = new MyWishlistsPage(driver);
+        driver.navigate().to(getProperty("url_my_wishlist_page"));
+        Assert.assertTrue(myWishlistsPage.isDisplayedViewMyShoppingCartBtn());
+
+        // add 3 different products to cart
+        for(int i = 1; i <= Integer.parseInt(Constants.TOTAL_PRODUCTS_IN_CART); i++){
+            driver.findElement(By.xpath("//*[@id='best-sellers_block_right']//li["+i+"]/a")).click();
+            Assert.assertTrue(myWishlistsPage.isDisplayedAddToCartBtn());
+            myWishlistsPage.clickAddToCartBtn();
+            myWishlistsPage.clickContinueShoppingBtn();
+            Assert.assertTrue(myWishlistsPage.isDisplayedAddToCartBtn());
+            driver.navigate().to(getProperty("url_my_wishlist_page"));
+            Assert.assertTrue(myWishlistsPage.isDisplayedViewMyShoppingCartBtn());
+        }
```

Comment:
test method shouldn't know anything about xpath.
Please move all logic related to product adding to the MyWishListsPage (or any other page that more suitable for this method)

Test(test method) should has as less logic as possible. It should carry out only about verifications